tests/src/lib.rs

```diff
@@ -697,13 +731,30 @@ mod utils {
     #[fixture]
     pub async fn hostname_service(#[future] kube_client: Client) -> KubeService {
         service(
-            kube_client,
             "default",
             "NodePort",
             "ghcr.io/metalbear-co/mirrord-pytest:latest",
             "hostname-echo",
             true,
             true,
+            kube_client,
+        )
+        .await
+    }
+
+    #[fixture]
+    pub async fn random_namespace_self_deleting_service(
+        #[future] kube_client: Client,
+    ) -> KubeService {
+        let namespace = format!("random-namespace-{}", random_string());
```

Comment:
Since this is offered here as a public test utility, it could be used by multiple tests, which could be executed simultaneously. Since the whole point is to let a test deploy its resources in an isolated namespace, the namespace's name cannot be constant. It either has to be random or "custom" (the calling test supplies a name for the namespace). Since the test doesn't actually care what the namespace is called, only that it's new, I opted for randomness, instead of making the test choose a name it doesn't care about. Moreover, parameterized tests (with `#[case]` or `#[values]`) would have to write a bit more code to have a different namespace name in each of their cases. I think this fixture is easier to use this way.

Reply:
Alright.